packages/esm-patient-chart-app/src/index.ts

```diff
@@ -90,6 +90,14 @@ function setupOpenMRS() {
           moduleName,
         }),
       },
+      {
+        id: 'notification-buttons-slot',
```

Comment:
The id of an extension should not end with `slot`. Only slots end with `-slot`.

Reply:
will make that change soon